ament_package/package.py

```diff
@@ -124,11 +153,17 @@ def validate(self):
             errors.append("Package name '%s' does not follow naming "
                           'conventions' % self.name)
 
+        version_regexp = '^[0-9]+\.[0-9_]+\.[0-9_]+$'
```

Comment:
Should this be exposed as a constant for sharing with consuming code? So it doesn't necessarily need to be.

Reply:
It was internal before this patch so I won't consider it part of this PR in order to keep the diff as small as possible.